identity/client/src/pages/groups/modals/AddModal.tsx

```diff
@@ -5,50 +5,78 @@
  * Licensed under the Camunda License 1.0. You may not use this file
  * except in compliance with the Camunda License 1.0.
  */
+
 import { FC, useState } from "react";
 import { InlineNotification } from "@carbon/react";
 import { FormModal, UseModalProps } from "src/components/modal";
 import useTranslate from "src/utility/localization";
 import { useApiCall } from "src/utility/api/hooks";
 import TextField from "src/components/form/TextField";
 import { createGroup } from "src/utility/api/groups";
+import { isValidGroupId } from "./isValidGroupId";
 
 const AddModal: FC<UseModalProps> = ({ open, onClose, onSuccess }) => {
-  const { t } = useTranslate();
+  const { t } = useTranslate("groups");
 
   const [callAddGroup, { loading, error }] = useApiCall(createGroup, {
     suppressErrorNotification: true,
   });
 
-  const [name, setName] = useState("");
+  const [groupName, setGroupName] = useState("");
+  const [groupId, setGroupId] = useState("");
+  const [description, setDescription] = useState("");
+  const [isGroupIdValid, setIsGroupIdValid] = useState(true);
 
   const handleSubmit = async () => {
     const { success } = await callAddGroup({
-      name: name.trim(),
+      name: groupName.trim(),
     });
 
     if (success) {
       onSuccess();
     }
   };
 
+  const validateGroupId = (id: string) => {
+    setIsGroupIdValid(isValidGroupId(id));
+  };
+
   return (
     <FormModal
       open={open}
-      headline={t("Create group")}
+      headline={t("createGroup")}
       onClose={onClose}
       onSubmit={handleSubmit}
       loading={loading}
-      loadingDescription={t("Adding group")}
-      confirmLabel={t("Create group")}
+      loadingDescription={t("creatingGroup")}
+      confirmLabel={t("createGroup")}
+      submitDisabled={!groupName || !groupId || !isGroupIdValid}
```

Comment:
I saw in the designs that there is a helperText that says:
"The group ID must be unique and cannot be modified. "
Do we want to add that?